docs/_static/auth/v2.yaml

```diff
@@ -545,19 +573,22 @@ components:
         scope:
           type: string
           description: granted scopes
-        presentation_definitions:
-          $ref: '#/components/schemas/RequiredPresentationDefinitions'
-        presentation_submissions:
-          type: object
-          additionalProperties:
-            $ref: '#/components/schemas/PresentationSubmission'
-          description: |
-            Mapping of Presentation Definition IDs that were fulfilled to Presentation Submissions.
-        vps:
-          type: array
-          items:
-            $ref: '#/components/schemas/VerifiablePresentation'
-          description: The Verifiable Presentations that were used to request the access token using the same encoding as used in the access token request.
+    ExtendedTokenIntrospectionResponse:
+      allOf:
```

Comment:
How's your Java client holding up?

Reply:
I'm getting it ready for v6, I'll make sure to test it.

Reply:
Tested, had to define VerifiablePresentation as `string` (JWT) or `object` (JSON-LD).